frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-node-manager/rpc_commit/readme.md

```diff
@@ -86,82 +93,26 @@ support 'candidate' datastore.
 
 ### Successful Example
 
-RPC commit input has 2 target nodes and the output describes the result
-of the commit.
+Nodes 'R1' and 'R2' has been changed. RPC commit input invoke all
```

Comment:
UniConfig commits nodes 'R1' and 'R2' that has been changed in the actual transaction.

Reply:
Done.